site/src/component/AppHeader/Profile.tsx

```diff
@@ -44,98 +39,90 @@ const Profile = () => {
+    setTheme(tab as 'light' | 'system' | 'dark');
+  };
+
+  const ProfilePopover = (
+    <Popover id="profile-popover" className="ppc-popover">
+      <div className="popover-body">
+        <div className="profile-popover__header">
+          <Image src={picture} alt={name} width="50" height="50" />
+          <div>
+            <h1 title={name}>{name}</h1>
+            <h2 title={email}>{email}</h2>
+          </div>
         </div>
-      </div>
-      <div className="profile-popover__links">
-        <ul>
-          <li>
-            <Link
+        <div className="profile-popover__theme-section">
+          <h4>Theme</h4>
+          <TabSelector tabs={themeTabs} selectedTab={getCurrentTheme()} onTabChange={handleThemeChange} />
+          <Divider />
+        </div>
+        <List className="profile-popover__links">
+          <ListItem>
+            <ListItemButton
               className={'profile-popover__link' + (pathname === '/reviews' ? ' active' : '')}
               href="/reviews"
               onClick={() => setShow(false)}
+              component={Link}
             >
-              <div>
+              <ListItemIcon>
                 <StickyNote2OutlinedIcon />
-              </div>
-              Your Reviews
-            </Link>
-          </li>
-          <li>
-            <button className="theme-button profile-popover__link" onClick={() => setTab('theme')}>
-              <div>{usingSystemTheme ? <LaptopIcon /> : darkMode ? <DarkModeIcon /> : <LightModeIcon />}</div>
-              Theme
-            </button>
-          </li>
-          <li>
-            <a href={'/api/users/auth/logout'} className="profile-popover__link">
-              <div>
+              </ListItemIcon>
+              <ListItemText primary="Your Reviews" />
+            </ListItemButton>
+          </ListItem>
+          {isAdmin && (
+            <>
+              <ListItem>
+                <ListItemButton
+                  className={'profile-popover__link' + (pathname === '/admin/verify' ? ' active' : '')}
+                  href="/admin/verify"
+                  onClick={() => setShow(false)}
+                  component={Link}
+                >
+                  <ListItemIcon>
+                    <GradingIcon />
+                  </ListItemIcon>
+                  <ListItemText primary="Verify Reviews" />
+                </ListItemButton>
+              </ListItem>
+              <ListItem>
+                <ListItemButton
+                  className={'profile-popover__link' + (pathname === '/admin/reports' ? ' active' : '')}
+                  href="/admin/reports"
+                  onClick={() => setShow(false)}
+                  component={Link}
+                >
+                  <ListItemIcon>
+                    <FlagIcon />
+                  </ListItemIcon>
+                  <ListItemText primary="View Reports" />
+                </ListItemButton>
+              </ListItem>
+            </>
+          )}
```

Comment:
I'd recommend pulling this out into a helper component in the same file `function AdminProfileLinks();`